Xamarin.Forms.ControlGallery.Android/ChangeDensityButtonRenderer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Xamarin.Forms;
+using Xamarin.Forms.Controls;
+using Xamarin.Forms.ControlGallery.Android;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(ChangeDensityButton), typeof(ChangeDensityButtonRenderer))]
+namespace Xamarin.Forms.ControlGallery.Android
+{
+	public class ChangeDensityButtonRenderer : Xamarin.Forms.Platform.Android.FastRenderers.ButtonRenderer
```

Comment:
I don't think adding a renderer just for a control you're using in the control gallery is something we would like to do

Reply:
I based on ApiLabel, to get some api (native) information and help-us to create some tests...
I think this can be util for another tests...


[ApiLabel](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.ControlGallery.Android/ApiLabelRenderer.cs)

Reply:
Like above: my bad, I thought this was in the regular Android project and not in the control gallery one :)

Reply:
Its OK, 😄